app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -82,6 +89,20 @@ class EventService(
         }
     }
 
+    fun getSpeakerByEventId(id: Long): Single<List<Speaker>> {
+        return speakerApi.getSpeakerforEvent(id)
+    }
+
+    fun getSpeakerByEventIdLiveData(id: Long): Single<List<Speaker>> {
+        return speakerApi.getSpeakerforEvent(id).flatMap { speakerList ->
```

Comment:
Use `doOnNext`

Reply:
i couldn't figure out this part as doOnNext works with flowable and having livedata for speakerwithEvent i'm bit confused about this part